csrankings-e.csv

```diff
@@ -728,6 +730,7 @@ Evangelos P. Markatos,University of Crete,https://www.csd.uoc.gr/CSD/index.jsp?c
 Evangelos Papapetrou,University of Ioannina,https://cs.uoi.gr/~epap,R6Ldg5UAAAAJ
 Evangelos Pournaras,University of Leeds,https://eps.leeds.ac.uk/computing/staff/6446/dr-evangelos-pournaras,EffH1CEAAAAJ
 Evangelos Sakkopoulos,University of Piraeus,https://www.unipi.gr/unipi/en/sakkopul.html,gLzYZ9wAAAAJ
+Evangelos A. Theodorou,Georgia Instutute of Technology,https://ae.gatech.edu/people/evangelos-theodorou,dG9MV7oAAAAJ

```

Comment:
There is a typo in the institution's name. It should be "Georgia **Institute** of Technology" instead of "Georgia **Instutute** of Technology"